snuba/settings/__init__.py

```diff
@@ -149,6 +149,12 @@
 # Metric Alerts Subscription Options
 ENABLE_SESSIONS_SUBSCRIPTIONS = os.environ.get("ENABLE_SESSIONS_SUBSCRIPTIONS", False)
 
+# The mock write latency to be applied when "writing" with the mock writer.
+# This is expressed in milliseconds
+MOCK_WRITER_LATENCIES = {"default": 100}
+# Deviation to apply to the above. In milliseconds
+MOCK_LATENCY_STD_DEVIATTION = 50
+
```

Comment:
I'd question the usefulness of putting this in settings. I think settings ideally is fairly permanent and is mostly useful so we can run Snuba differently across different environments. In this case, it seems more these settings support a test we are only planning to run in prod and nowhere else. I think it'd be better to just pass the values on the CLI and not pollute settings (and possibly confuse operators of Snuba)

Reply:
Isn't it worse if we add these permanent parameters to the command line interface that are only usable if another option is set to true and during a test?

Reply:
Btw, when I wrote this comment I sort of assumed there was a separate API for the mock test, not just an option on the regular consumer, hence not concerned about adding additional options. Any thoughts on also doing that?

Reply:
I think we we typing at the same time. Yeah I assumed it was a separate command.. any downside to making it so?

Reply:
I did think about that. IT seemed it would introduce a massive duplication. But I think it may have been only relevant for the multi storage consumer. Let me try again.

Reply:
What is the use case for ever needing to load test the multistorage consumer?

Reply:
If a consumer in production runs as a multistorage one (like the metrics one) we need to load test it in the same way

Reply:
I removed the settings, and also the support for the multistorage consumer. 
Will take care of that in another PR because of the amount of refactoring needed. We are only using it for Change data capture as of now